src/elements/content-uploader/ModernizedUploadsManagerPanel.scss

```
@@ -0,0 +1,47 @@
+@keyframes bcu-modernized-slideIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "bcu-modernized-slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)